scripts/github-repos/README.md

```diff
@@ -1,53 +1,138 @@
-# Bulk creation/deletion of many repos and cs169a-team
+# Bulk creation/deletion of many repos and student teams
 
 ```
-Usage: ./github-repos.rb [required options] [invite|repos|remove|remove_access]
+Usage: ./github-repos.rb [required options] [invite|create_teams|indiv_repos|team_repos|remove_indivi_repos|remove_team_repos|remove_teams|remove_team_repo_access] [optional options]
 
 GITHUB_ORG_API_KEY for the org must be set as an environment variable.
 
-'invite' invites students provided in .csv file and creates teams, 'repos' creates team repos, 'remove' remove students, repos, teams from the org
-
 It's safe to run multiple times.
 
-Required arguments:
-    -c, --csv=CSVFILE                CSV file containing at least "Team" and "Email" named columns
-    -o, --orgname=ORGNAME            The name of the org eg org_name
-    -f, --filename=FILENAME          The base filename for repos, eg "fa23-actionmap-04", actionmap is the base file name of the repo
-    -p, --prefix=PREFIX              Semester prefix, eg "fa23" create a repos prefix, "fa23-actionmap-04", etc.
-    -t, --template=TEMPLATE          The repo name within the org to use as template eg repo_name (Assume the repo own by org)
+'invite':
+Create a team called STUDENTTEAM under the org that has all students in the same semester and send invitation
+If STUDENTTEAM is already exist, the script will resent invitation to students. 
+(Temporary for special situations in Fa23)
+If PREFIX is provided, it will assume the csv file contains "Team" column, and create child teams under 
+the STUDENTTEAM, and invites them to child teams.
+
+Required options:
+    -c, --csv=CSVFILE                CSV file containing at leaset "Username" named columns
+    -s, --studentteam=STUDENTTEAM    The team name of all the students team
+    -o, --orgname=ORGNAME            The name of the org
+
+Optional options:
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+
+'create_teams' (Not functioning)
+Assuming students are in STUDENTTEAM, create child teams for students for CHIP 10.5 and add them to the child team.
+
+Required options:
+    -c, --csv=CSVFILE                CSV file containing at least "Team" and "Username" named columns
     -s, --studentteam=STUDENTTEAM    The team name of all the students team

```

Comment:
i don't understand the documentation.  is STUDENTTEAM a single large team containing all students in the class, or is each STUDENTTEAM a small team that shares a repo?  i think TEAM is not a good name for these options if it means the first thing, since the word "Team" in the csvfile (previous option) means something completely different.